sso-kit-core/pom.xml

```diff
@@ -25,9 +25,23 @@
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-oauth2-jose</artifactId>
         </dependency>
+        <!-- We need to exclude json-smart and bump to 2.4.10 until a new 
+             spring version is released -->
         <dependency>
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-config</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.minidev</groupId>
+                    <artifactId>json-smart</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
+            <version>2.4.10</version>
+        </dependency>
```

Comment:
Redundant closing `</dependency>` tag is causing validation to fail:
```suggestion
```

Reply:
oh, thanks..